ring-jetty-adapter/test/ring/adapter/test/jetty.clj

```diff
@@ -200,3 +201,38 @@
               (Thread/sleep 250)
               (recur (inc i))))
           (is (= thread-count (count (all-threads)))))))))
+
+(deftest test-ssl-context-factory
+  (defn- get-ssl-context-factory
```

Comment:
Can you move these `defn-` forms out of the `deftest`? It's generally not a good idea to have `def`s inside a function, since all it does is ensure that the statements are executed more than once.